### PR TITLE
Fix play button action

### PR DESCRIPTION
**Pull request by ChuckBTaylor**

### CODE REVIEW

File: src/components/organisms/modal/VideoModal.tsx [hunk 1]
```diff
@@ -1,32 +1,65 @@
...
+function getMoreImportantScreenValue(
+  height: number,
+  width: number
+): 'width' | 'height' {
+  return width / height > sixteenByNineRatio ? 'height' : 'width'
+}
 
 export function VideoModal() {
+  const { innerHeight: height, innerWidth: width } = window
+  const [, setWindowWidth] = useState(width)
```
Comment (mwilkins74) [inline src/components/organisms/modal/VideoModal.tsx:18]: Assuming leaving the state variable out here was unintentional?
```suggestion
  const [windowWidth, setWindowWidth] = useState(width)
```

**Review by mwilkins74 — APPROVED** [review]
One minor fix. Otherwise good to go!

File: src/components/organisms/modal/VideoModal.tsx (hunk 1 above)
Comment (ChuckBTaylor) [reply src/components/organisms/modal/VideoModal.tsx:18]: Nope. Was intentional.  Many linters (I'm pretty sure Netlify is one of them) will not be happy if you declare unused variables.  We don't use `windowWidth` so Netlify will fail the build with that lint error.

One of the common ways to indicate that a variable is unused is to prefix it with a `_`  e.g.: `_windowWidth`.  I think this takes some amount of configuring on the linter side, hwoever.

The other way to indicate a variable is unused, is not to declare the variable at all (which is what we did here).

Comment (mwilkins74) [reply src/components/organisms/modal/VideoModal.tsx:18]: That is what I thought was going on here. I have never actually seen it implemented so I aired on the side of caution (?) and flagged it for fixing. 

Appreciate the explanation and it makes perfect sense!

**Pull request merged**